skimage/registration/_phase_cross_correlation.py

```diff
@@ -178,6 +178,11 @@ def _disambiguate_shift(reference_image, moving_image, shift):
         if corr > max_corr:
             max_corr = corr
             max_slice = test_slice
+    if max_slice is None:
+        warnings.warn(
+            'Phase cross correlation shift disambiguation inconclusive', stacklevel=4
```

Comment:
Thanks for the update. One last suggestion otherwise this looks good IMO:

```python
f"Could not determine real-space shift for the periodic shift {shift!r}"
```
I think this message might be more helpful to users.

Reply:
Is there a way to include the word "disambiguate" in the message? (Since that is what the user requested and failed.)

Reply:
Is there a way to include the word "disambiguate" in the message? (Since that is what the user requested and failed.)

Reply:
Maybe replace "determine" with "disambiguate". But "disambiguate" feels like a possible language barrier to non-native speakers so I like the following version more:

```python
f"Could not determine real-space shift for the periodic shift {shift!r} "
f"as requested with `disambiguate=True`"
```

What do you think?

Reply:
> But "disambiguate" feels like a possible language barrier to non-native speakers

:-1: it's not jargon, it doesn't sound pompous or pedantic, so I don't see why it should be avoided (*dis-ambiguate*, the Latin root is transparent so it's not... ambiguous at all :wink:)

Reply:
Oh, I agree with your assessment. Just that I'd add that it may be advanced vocabulary which is entirely subjective. :)